src/App.tsx

```diff
@@ -1,13 +1,22 @@
-import {Button} from "./ui/Button/Button"
-import {Icon} from "./ui/icon/Icon"
+import { Input } from "./ui/input/Input"
+import { Icon } from "./ui/icon/Icon"
 import "./scss/style.scss"
+import { useState } from "react"
 
 
 function App() {
+    const [value, setValue] = useState("")
+    console.log(value)
+    
     return (
-        <Button prefixIcon={<Icon name="filter"/>}>
-            Вход
-        </Button>
+        <Input 
+            placeholder="Username" 
+            type="password"
+            value={value}
+            onChange={setValue}
+            iconPassword={<Icon name="password" className="icon-input"/>}
+            clearable={<Icon name="close" className="icon-input"/>}
```

Comment:
```suggestion
            clearable
```

`type clearable = boolean`

Reply:
исправлено